src/main/java/Event.java

```diff
@@ -0,0 +1,30 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.text.DateFormat;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.time.format.FormatStyle;
+import java.util.Date;
+
+
+class Event extends Task {
+
+    private final LocalDate time;
+
+    public Event(String name, LocalDate time) {
+        super(name);
+        this.time = time;
+    }
+
+    @Override
+    public String toString() {
+        if (super.getStatus() == 1) {
+            return "[E][X] " + super.getName() + " (at: "
+                    + time.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL)) + ")\n";
+        } else {
+            return "[E][ ] " + super.getName() + " (at: "
+                    + time.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL)) + ")\n";
+        }
```

Comment:
perhaps instead of repeating the return string twice to handle the different status of the Task, you can first do something like:

```java
public String toString() {
        String statusString = super.getStatus() == 1 ? "[E][X]" : "[E][ ]";
        return statusString + super.getName() + " (at: "
                    + time.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.FULL)) + ")\n";
        

```

what do you think 🤔 ?